core-js/src/main/javascript/cdf/dashboard/Dashboard.js

```diff
@@ -163,19 +167,25 @@ define([
     },
 
     /**
-     * Determines if components and params are retrieved
-     * from the _window_ object or from the dashboard instance
+     * @summary The {@link cdf.dashboard.RefreshEngine|RefreshEngine} that manages the component refresh cycle.
+     * @description The {@link cdf.dashboard.RefreshEngine|RefreshEngine} that manages the component refresh cycle.
+     *
+     * @protected
+     * @type {cdf.dashboard.RefreshEngine}
+     */
+    refreshEngine: undefined,
+
+
+    /**
+     * @summary Determines if components and params are retrieved from the _window_ object or from the dashboard instance
+     * @description Determines if components and params are retrieved from the _window_ object or from the dashboard instance
```

Comment:
replace "params" with "parameters", " _window_" with " `window` " and add punctuation.